llama_stack/distribution/start_stack.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
curious why the other scripts don't use this?

Reply:
I’m not sure why this wasn’t included, but since we’re making multiple changes, we might as well add it. It improves portability across different Unix-like systems.